### ISSUE TITLE
Installation via conda package manager

### ISSUE THREAD

**Issue by johngull** [aleju/imgaug#320]
It will be great to be able to install imgaug via conda package manager.
It is very popular now and has some advantages over pip.

**Comment by mdraw** [comment]
I'm on it: https://github.com/conda-forge/staged-recipes/pull/8504

**Comment by mdraw** [comment]
You can now install imgaug from conda-forge: https://github.com/conda-forge/imgaug-feedstock.
I think this issue can be closed now.

**Comment by johngull** [comment]
Before closing issue, please add exact installation command to the README.

**Comment by aleju** [comment]
Thanks for the help @mdraw !
I see in https://github.com/conda-forge/imgaug-feedstock/blob/master/recipe/meta.yaml that the recipe contains a list of the current dependencies. How to update these when they change? I guess it doesn't happen automatically.

**Comment by mdraw** [comment]
@aleju Indeed, the conda feedstock maintainers (currently only me, but I can add you if you want) have to keep dependencies up to date manually.
When you upload a new version to PyPI, a PR to update the conda package will be automatically made by a bot, but it has to be reviewed and merged by a human. Dependency changes can be made by maintainers inside these PRs.
To be on the safe side, it's also possible to add the automated tests from `imgaug/test/` to the `test` section of the `meta.yaml`, but that would require the xdoctests package, which is not yet available on conda-forge.

**Comment by aleju** [comment]
@mdraw Would be great if you could add me as a maintainer. Not very likely that there will be dependency changes in the near future, but that is always hard to predict.
And the tests could probably be run without xdoctests for conda.

**Comment by aleju** [comment]
Install commands for conda are now in the readme.

**Comment by juanluisrosaramos** [comment]
I 'm installing a fresh new desktop pc with Ubuntu 19.04 and after installing imaug with conda the import (in python) gives me an error
~/anaconda3/envs/tfgpu/lib/python3.6/site-packages/scipy/sparse/linalg/interface.py in <module>
     48 
     49 from scipy.sparse import isspmatrix
---> 50 from scipy.sparse.sputils import isshape, isintlike, asmatrix
     51 
     52 __all__ = ['LinearOperator', 'aslinearoperator']

**ImportError: cannot import name 'asmatrix'**
